A14-channelz.md

```diff
@@ -684,12 +732,21 @@ message ChannelRef {
   reserved 3, 4, 5, 6;
 }
 
+message SubchannelRef {
+  // The globally unique id for this subchannel.  Must be a positive number.
+  int64 subchannel_id = 7;
+  // An optional name associated with the subchannel.
+  string name = 8;
+  // Intentionally don't use field numbers from other refs.
```

Comment:
Why does this matter?

Reply:
It doesn't but I like having the refs be distinguishable by the tag numbers.